src/actionExecutor.ts

```diff
@@ -39,14 +39,14 @@ export class ActionExecutor {
     let result: Result<any, Error>;
 
     const triggerResult = await this.triggerRequestFromAction(action);
-    logger.trace(`Response from action: ${JSON.stringify(triggerResult)}`);
+    logger.log("trace", `Response from action`, triggerResult);
```

Comment:
Can we implement a helper function that wraps `logger.log("trace", "awesome loge message")` for this to reduce the number of characters (assuming trace level logging is a common use case).  Something like `traceLog("even awesomer log message")`?